From 26fee64a01dc8d809a3b06274440810a6d569175 Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.com>
Date: Mon, 26 Feb 2024 18:00:53 +0000
Subject: [PATCH 0912/1085] dtoverlays: Correct width/height on Waveshare 2.8"
 panel

Width and height were swapped, so touch didn't work over the whole
panel.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
---
 .../boot/dts/overlays/vc4-kms-dsi-waveshare-panel-overlay.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/arch/arm/boot/dts/overlays/vc4-kms-dsi-waveshare-panel-overlay.dts
+++ b/arch/arm/boot/dts/overlays/vc4-kms-dsi-waveshare-panel-overlay.dts
@@ -77,8 +77,8 @@
 
 	__overrides__ {
 		2_8_inch = <&panel>, "compatible=waveshare,2.8inch-panel",
-				   <&touch>, "touchscreen-size-x:0=640",
-				   <&touch>, "touchscreen-size-y:0=480",
+				   <&touch>, "touchscreen-size-x:0=480",
+				   <&touch>, "touchscreen-size-y:0=640",
 				   <&touch>, "touchscreen-inverted-y?",
 				   <&touch>, "touchscreen-swapped-x-y?";
 		3_4_inch = <&panel>, "compatible=waveshare,3.4inch-panel",
